### PR TITLE
Masterblade

### PR DESCRIPTION
**Pull request by Kenini1805**

### CODE REVIEW

File: public/css/flat-icon/flaticon.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	url("flaticon.ttf") format("truetype"),
+	url("flaticon.svg") format("svg");
+	font-weight: normal;
+	font-style: normal;
+}
+[class^="flaticon-"]:before, [class*=" flaticon-"]:before,
+[class^="flaticon-"]:after, [class*=" flaticon-"]:after {   
+	font-family: Flaticon;
+        font-size: 25px;
+font-style: normal;
```
Comment (trinhductoan) [inline public/css/flat-icon/flaticon.css]: indent

File: public/css/flat-icon/flaticon.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
+@font-face {
+	font-family: "Flaticon";
+	src: url("flaticon.eot");
+	src: url("flaticon.eot#iefix") format("embedded-opentype"),
+	url("flaticon.woff") format("woff"),
+	url("flaticon.ttf") format("truetype"),
+	url("flaticon.svg") format("svg");
+	font-weight: normal;
```
Comment (trinhductoan) [inline public/css/flat-icon/flaticon.css]: dùng 4 space thay cho 1 tab

File: resources/views/backend/master.blade.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<html class="no-js" lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>{{ config('app.name') }}</title>
+    {{ Html::style('assets/foundation/css/foundation.css') }}
+    {{ Html::style('assets/foundation-icon-fonts/foundation-icons.css') }}
+    <link rel="stylesheet" href="{{ URL::asset('/css/style.css') }}">
+    <link href="{{ URL::asset('/css/flat-icon/flaticon.css') }}" rel="stylesheet" />
```
Comment (trinhductoan) [inline resources/views/backend/master.blade.php]: bổ sugn 1 section cho `styles`

File: resources/views/backend/master.blade.php [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+            @include('backend.layout.top-menu')
+            <div class="row">
+                @include('backend.layout.left-side-bar')
+                @yield('main')
+            </div>
+        </div>
+    </div>
+    {{ Html::script('assets/modernizr/modernizr.js') }}
+    {{ Html::script('assets/jquery/dist/jquery.js') }}
+    {{ Html::script('assets/foundation/js/foundation/foundation.js') }}
```
Comment (trinhductoan) [inline resources/views/backend/master.blade.php]: bổ sung 1 section cho `scripts`

**Comment by Kenini1805** [comment]
ready

**Comment by trinhductoan** [comment]
OK

**Pull request merged**